cypress/integration/candidate.spec.js

```diff
@@ -94,7 +94,7 @@ const thenIShouldBeSignedInSuccessfully = () => {
 };
 
 const isBetweenCycles = () => {
-  const endOfCycle = +new Date(2020, 7, 24);
+  const endOfCycle = +new Date(2020, 7, 25);
```

Comment:
I thought these were the wrong month but they are correct the JS Date function takes a `monthIndex` so 7 === August etc. However the deadline for `endOfCycle` is really midnight on 24 August so I think we need to add one to the date.

Reply:
Ah yes, always forget about that!